docs/zh-CN/docs/sql-manual/sql-functions/string-functions/convert_to.md

```diff
@@ -0,0 +1,69 @@
+---
+{
+    "title": "convert_to",
+    "language": "zh-CN"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
```

Comment:
missing en doc.
and please refer to : https://doris.apache.org/zh-CN/community/how-to-contribute/contribute-doc/#%E6%96%87%E6%A1%A3%E5%A4%9A%E7%89%88%E6%9C%AC

to add version tag : `<version since="1.2">`